sway/criteria.c

```diff
@@ -228,8 +228,13 @@ static bool criteria_matches_view(struct criteria *criteria,
 
 		switch (criteria->app_id->match_type) {
 		case PATTERN_FOCUSED:
```

Comment:
Style nit: instead of using braces, can use a semicolon:

```c
case PATTERN_FOCUSED:;
	const char *focused_app_id = view_get_app_id(focused);
```

The C syntax is annoying here.

Reply:
Cheers, didn't know about that one.

Removed in favour of `lenient_strcmp` as suggested